.travis.yml

```diff
@@ -22,10 +25,18 @@ matrix:
     - os: osx
       osx_image: xcode10.1
 
+before_install:
+  - wget "https://www.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.zip"
+  - wget "https://www.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.zip.asc"
+  - wget https://www.apache.org/dist/maven/KEYS
+  - gpg --import KEYS
+  # Should be checking for GOODSIG but VALIDSIG is a workaround for the key being expired
+  - if [[ $(gpg --verify --status-fd 1 "apache-maven-${MAVEN_VERSION}-bin.zip.asc" "apache-maven-${MAVEN_VERSION}-bin.zip" 2>/dev/null | sed -n 's/\[GNUPG:\] VALIDSIG \([A-F0-9]\{40\}\) .* \([A-F0-9]\{40\}\)$/\1\n\2/p' |  wc -c) == 0 ]]; then exit 1; fi
+  - unzip -qq "apache-maven-${MAVEN_VERSION}-bin.zip"
+  - export PATH="${PWD}/apache-maven-${MAVEN_VERSION}/bin:${PATH}"
```

Comment:
As we chatted about on email, I'll take another look at the Maven Wrapper now that there are beta Windows builds on/for Travis (and the wrapper would work cross-platform).

Reply:
@cowwoc It won't be this weekend that I get to this. I'll probably get to it by the end of next weekend, but if you need an immediate fix (if the failed Travis build is holding up something else) you could merge in this Mac/Linux fix as a temporary workaround. Otherwise, you can wait for a fix that works with the new Windows beta + Mac/Linux (using the plugin).

Reply:
No worries. I'll wait. I'll stick the an older version of cmake for now.

Reply:
It's been updated to work with the Maven wrapper now. This _should_ work when Travis supports Java on Windows; they say they'll have Bash installed on the Windows machine to maintain compatibility with Linux and Mac OS builds.

The only way to get Java on Windows now would be to change the language to 'shell' instead of 'Java' and install the JDK manually. I'm inclined to wait for official support from Travis.